docs/api/alerts/create.asciidoc

```diff
@@ -9,7 +9,13 @@ Create {kib} alerts.
 [[alerts-api-create-request]]
 ==== Request
 
-`POST <kibana host>:<port>/api/alerts/alert`
+`POST <kibana host>:<port>/api/alerts/alert/<id>`
+
+[[alerts-api-create-path-params]]
+==== Path parameters
+
+`<id>`::
+  (Optional, string) Specifies a UUID to use instead of a randomly generated ID.
```

Comment:
Should this say something about how it has to be UUID v1 or v4?

Reply:
TIL there's a uuid v5

Does it actually matter which uuid version they provide us with? 🤔

Reply:
We are bound to whatever this regex allows: https://github.com/elastic/kibana/blob/f413957827b0294d12cf9e460d73c6ff26314fe8/src/core/server/saved_objects/service/lib/utils.ts#L28

But we only have tests for v1 and v4: https://github.com/elastic/kibana/blob/51cfa90dc5a387c88a3b66bfa04a3d6580614b5e/x-pack/test/alerting_api_integration/spaces_only/tests/alerting/create.ts#L114-L150

I will update the docs to add v1/v4.

Reply:
Fixed in https://github.com/elastic/kibana/pull/92410/commits/f8386b59eebc2a8e23e536a8e744c95a63a12fcf.